docs/whats_new.rst

```diff
@@ -43,6 +44,8 @@ Bug
 
 API
 ~~~
+- The permissible parameters for the following methods were removed and/or reordered: :func:`ransac.ransac_correlations`, :func:`ransac.run_ransac`, and :func:`ransac.get_ransac_pred` methods, by `Yorguin Mantilla`_ (`#51 <https://github.com/sappelhoff/pyprep/pull/51>`_)
+- The following methods were moved to a new module named :mod:`ransac`: :meth:`NoisyChannels.ransac_correlations`, :meth:`NoisyChannels.run_ransac`, and :meth:`NoisyChannels.get_ransac_pred` methods, by `Yorguin Mantilla`_ (`#51 <https://github.com/sappelhoff/pyprep/pull/51>`_)
```

Comment:
These methods should have been private to begin with, I think we should only expose "find bad by ransac" to the user, not all the other functions. The "internals" should be something for developers to care about, not the end user, because it's less confusing that way.

```suggestion
- The following methods have been moved to a new module named :mod:`ransac` and are now private: :meth:`NoisyChannels.ransac_correlations`, :meth:`NoisyChannels.run_ransac`, and :meth:`NoisyChannels.get_ransac_pred` methods, by `Yorguin Mantilla`_ (`#51 <https://github.com/sappelhoff/pyprep/pull/51>`_)
```